backend/src/routes/doctorOrders.js

```diff
@@ -177,6 +177,14 @@ router.patch('/api/updateRx/:id/pickedUp', async (req, res) => {
       }
     });
     console.log('Sent RxFill to EHR and received status from EHR', status.data);
+
+    const remsAdminStatus = await axios.post('http://rems-administrator:8090/ncpdp', rxFill, {
+      headers: {
```

Comment:
Should we expect the return to be xml on this one as well?

Reply:
The xml return is part of the NCPDP SCRIPT RxFill exchange. Even if we do nothing with it we should include the STATUS return to be complete.

Reply:
oh okay makes sense